device-types/Cisco/ISR-1127-8PLTEP.yaml

```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: Cisco
+model: ISR 1127-8PLTEP
+part_number: C1127-8PLTEP
+slug: cisco-isr-1127-8pltep
+weight: 4.53
+weight_unit: lb
+u_height: 1
+is_full_depth: false
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/1
+    type: 1000base-t
+  - name: GigabitEthernet0/1/0
+    type: 1000base-t
+  - name: GigabitEthernet0/1/1
+    type: 1000base-t
+  - name: GigabitEthernet0/1/2
+    type: 1000base-t
+  - name: GigabitEthernet0/1/3
+    type: 1000base-t
+  - name: GigabitEthernet0/1/4
+    type: 1000base-t
+  - name: GigabitEthernet0/1/5
+    type: 1000base-t
+  - name: GigabitEthernet0/1/6
+    type: 1000base-t
+  - name: GigabitEthernet0/1/7
+    type: 1000base-t
+module-bays:
+  - name: Cellular
+    position: '1'
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-micro-b
```

Comment:
Is this port actually used for console access? If so then this is fine, if not then please remove it.